packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -202,6 +207,7 @@ export default class Dropdown extends React.Component<Props, State> {
                 alignment={alignment}
                 items={this._getMenuItems()}
                 keyboard={this.state.keyboard}
+                style={this.props.style}
```

Comment:
This is another place where `style` could be destructured from this.props like `alignment` and the other props to be more consistent.

Also, I noticed that we're creating a `sharedProps` on line 185 but I don't see it being used anywhere.  Can you confirm this and if that's the case reform `sharedProps`?

Reply:
Good call. And for sharedProps yes its not being used so i got rid of it